index.html

```diff
@@ -2,10 +2,16 @@
 <html>
   <head>
     <title>JavaScript lodash Templates</title>
+    <script src="lodash.min.js"></script>
```

Comment:
Nit: could we move this to the bottom of `<body>` (above `<script src="index.js"></script>`)?